app/views/posts/index.html.erb

```diff
@@ -1 +1,36 @@
-<h1>All Posts will be shows here</h1>
\ No newline at end of file
+<div>
+  <div>
+        <%= image_tag @posts[0].author.photo, alt: "User pic" %>
+         <div>
+           <h1><%= @posts[0].author.name %></h1>
+           <p>Number of posts: <%= @posts[0].author.posts_counter %></p>
+         </div>
+  </div>
```

Comment:
- [ ] In continuation with the previous comment the code at line `3` was throwing an error after I added the URL of a random user image and that's because `image_tag` will try to look for the actual image in the asset pipeline but as I mentioned earlier since won't actually be uploading an image I'd suggest you use the normal `<img />` tag to render the user image.
- [ ] Please keep in mind this comment also applies to other places in your codebase where you're using the `image_tag`.
- [ ] After making the changes make sure the layout matches the one shown in the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAACJTIBN6AVJIRQUUN7D5B6ZWZKE4YKA)

Reply:
Updated with <img /> tag